main.py

```diff
@@ -51,6 +51,7 @@ async def redirect():
         return RedirectResponse(url="/admin/#new")
 
 
+app.mount("/api/data", StaticFiles(directory="data"), name="static")
```

Comment:
@leehuwuj just a note: this will change with the next create-llama version to /api/files/data